edr_server/core/models/parameters.py

```diff
@@ -49,10 +49,13 @@ def _prepare_json_for_init(cls, json_dict: JsonDict) -> JsonDict:
         if "symbol" in json_dict and isinstance(json_dict["symbol"], dict):
             json_dict["symbol"] = Symbol.from_json(json_dict["symbol"])
 
-        if "labels" in json_dict and isinstance(json_dict["label"], dict):
```

Comment:
This allows us to use `from_json` on the Parameter Unit Labels, was receiving the error `InvalidEdrJsonError: Error during conversion from JSON: init() got an unexpected keyword argument 'label'`